apps/ledger-live-desktop/tests/specs/speculos/subAccount/receive.subAccount.spec.ts

```diff
@@ -0,0 +1,47 @@
+import test from "../../../fixtures/common";
+import { Token } from "../../../enum/Tokens";
+import { specs } from "../../../utils/speculos";
+import { Application } from "tests/page";
+import { addTmsLink } from "tests/fixtures/common";
+
+const tokens: Token[] = [
+  Token.ETH_USDT,
+  Token.ETH_LIDO,
+  Token.TRON_USDT,
+  Token.TRON_BTT,
+  Token.BSC_BUSD,
+  Token.BSC_SHIBA,
+  Token.MATIC_DAI,
+  Token.MATIC_UNI,
+];
+
+//Reactivate test after fixing the GetAppAndVersion issue - Jira: LIVE-12581
+for (const [i, token] of tokens.entries()) {
+  test.describe.skip("Receive @smoke", () => {
+    test.use({
+      userdata: "speculos-subAccount",
+      testName: `receive subAccount_${token.tokenName}`,
+      speculosCurrency: specs[token.parentAccount.currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: i,
+    });
+
+    test(`[${token.tokenName}] Receive Sub Account (${token.tokenNetwork})`, async ({ page }) => {
+      addTmsLink(["B2CQA-640"]); //TODO: create a new Jira Ticket for this scenario
```

Comment:
could you please ask someone of manual QA for a test case ticket? also, do we actually need to automate receive flow for tokens? there was no such test case in the regression scope

Reply:
done